src/Microsoft.Health.Fhir.Core/Features/Definition/SupportedSearchParameterDefinitionManager.cs

```diff
@@ -74,7 +75,7 @@ public SearchParameterInfo GetSearchParameter(Uri definitionUri)
 
         public IEnumerable<SearchParameterInfo> GetSupportedButNotSearchableParams()
         {
-            return _inner.AllSearchParameters.Where(p => p.IsSearchable == false && p.IsSupported == true);
+            return AllSearchParameters.Where(p => p.IsSearchable == false || p.SortStatus == SortParameterStatus.Supported);
```

Comment:
Is the case of IsSearchable == false, and SortStatus == NotSupported and p.IsSupported == true, possible?

Reply:
In CosmosDB, SortStatus will only selectively be enabled on Search parameters we can both index and types we can sort on

Reply:
Yep, that makese sense.  Not all parameters are sortable. 
A SearchParameter which is supported, but not sortable would have the following properties:
p.IsSearchable == false
p.IsSupported == true
p.SortStatus == SortParameterStatus.Disabled
In which case I believe it should be included in the returned list of params, but as is would not be.

Also, we may have the case that we have a parameter which is possible to sort, and also already Searchable:
p.IsSearchable == true
p.SortStatus == Supported
This parameter would be included in the results, but should not be.

Reply:
Discussed with Brendan, and realized that I missed the change to using the AllSearchParams rather than _inner. which applies the IsSupported filter.  Looks good.